app/views/queries/_index_sort_bar.html.haml

```diff
@@ -10,17 +10,20 @@
     .col-md-4
       %div#author
         %strong
-          %a{ 'ng-click' => 'queryIdxCtrl.pagination.setSort("author")', 'title' => 'Sort By Author' }
+          %a{ 'ng-click' => 'queryIdxCtrl.pagination.setSort("author")', 'title' => 'Sort By Author',
+              'class' => 'group-query-items' }
```

Comment:
our style with classes is usually to use `.` notation -- so like `%a.group-query-items`